schematic/models/validate_manifest.py

```diff
@@ -21,14 +21,7 @@
 from schematic.schemas.generator import SchemaGenerator
 from schematic.store.synapse import SynapseStorage
 from schematic.models.GE_Helpers import GreatExpectationsHelpers
-
-from ruamel import yaml
```

Comment:
Were the imports below unused?

Reply:
That's correct. They were leftovers from before `GE_Helpers` was added, and were only used in those functions. Since it's been moved to its own file they're no longer necessary here, but they are present where needed in the other file.